### ISSUE TITLE
feat: include different Qwik starters

### ISSUE THREAD

**Issue by dmitry-stepanenko** [qwikifiers/qwik-nx#27]
Be able to scaffold different starters in the same manner it's done using Qwik CLI https://qwik.builder.io/docs/getting-started/

**Comment by DominikPieper** [comment]
I will take this one

**Comment by dmitry-stepanenko** [comment]
@DominikPieper great thank you 🙂 Feel free to reach out if you have any questions

**Comment by DominikPieper** [comment]
@dmitry-stepanenko I added the two options for app and documentation. Any more options? I do some experiments and have to write some unit tests after that you could have a look

**Comment by dmitry-stepanenko** [comment]
@DominikPieper the "app" is an existing default one, right? I think it should also be convenient to get an "empty" preset without any pre-configured layouts, logic, flower component etc.
Also I've just noticed that Qwik has added a "todolist" component to their "app" starter, can you please copy it over?

**Comment by DominikPieper** [comment]
@dmitry-stepanenko yes that's the default one. I'll take a look at that. Saw that they added something here
